client/my-sites/domains/domain-management/dns/txt-record.jsx

```diff
@@ -24,17 +22,6 @@ class TxtRecord extends Component {
 
 		if ( value?.length === 0 ) {
 			return translate( 'TXT records may not be empty' );
-		} else if ( value?.length > 255 ) {
```

Comment:
I just wonder if we should still put some sensible retriction on the TXT record length. Other than that I tested it and it seems to work well enough

Reply:
Done.  I set it to 65535.